### DISCUSSION TITLE
Regelung der Wallbox GOECharger

### DISCUSSION THREAD

**Discussion by modulder** [camueller/SmartApplianceEnabler#386] [Allgemein]
Hallo,
Ich habe die Installation über meinen Raspy mit SAE 1.6.21 installiert.
Läuft soweit auch schon, nur die Regelung der Wallbox funktioniert leider nicht. Diese lädt quasi immer mit 3,8KW, auch wenn als Beispiel nur 1KW über die PV rein kommt.

Vielen Dank für das tolle Programm und die Unterstützung, sicherlich habe ich nur eine Kleinigkeit übersehen.

Danke und Grüße
Michael

**Comment by camueller** [comment]
Von welcher Art des Ladens sprichst Du? Beim Überschussladen wird die Leistung vom SHM vorgegeben. Beim Sofortladen wird mit max. Stromstärke geladen.
